core/domain/interaction_jobs_one_off.py

```diff
@@ -152,6 +182,36 @@ def reduce(key, values):
         yield (key, values)
 
 
+class ItemSelectionInteractionLtOneOffJob(
```

Comment:
You can audit this in the job above this one, also this needs to be finished and tested pretty soon since we are doing a release cut tomorrow and after that there will be a release without jobs.